stdlib/test/builtin/test_print.mojo

```diff
@@ -73,7 +74,17 @@ fn test_print_sep():
     print("a", 1, 2, sep="/", end="xx\n")
 
 
+fn test_print_stderr():
+    # TODO: check print to stderr
```

Comment:
Do you have ideas on how you'd like to test that we wrote to stderr here?

Reply:
If I could offer a suggestion: since this test uses FileCheck anyway, you could add a check specifically for this. Something like this:

```
# RUN: %mojo %s 2>&1 1>/dev/null | FileCheck %s --check-prefix=CHECK-STDERR
# CHECK-STDERR: stderr
```

The `2>&1 1>/dev/null` should redirect all the `print` statements from other tests to `/dev/null`, silencing them, while the `print(..., file=sys.stderr)` statements should be printed to `stdout`, which `FileCheck` makes expectations against. As a result, the `CHECK-STDERR: stderr` expectation would fail if `stderr` was only being printed to `stdout`, not `stderr`.